Dockerfile

```diff
@@ -4,13 +4,12 @@
 # <NODE>
 ADD tools/build /stack/boilerplate
 
-ENV NODE_VERSION=6.10.1 \
-    NPM_VERSION=3.10.10
-
-RUN bash /stack/boilerplate/install.sh
+ENV NODE_VERSION=8.9.4 \
+    NPM_VERSION=5.4.0
 
+RUN sh /stack/boilerplate/install.sh
 ENV NODE_PATH=$NVM_DIR/versions/node/v$NODE_VERSION/lib/node_modules \
```

Comment:
so this is still required? it's also in install.sh

Reply:
It sticks here, it doesn't stick in install. 

It's not useful on the Alpine build, but mandatory on Debian

Reply:
i see :(